packages/waku/src/lib/middleware/dev-server-impl.ts

```diff
@@ -111,7 +111,11 @@ const createMainViteServer = (
         fsRouterTypegenPlugin(config),
       ],
       optimizeDeps: {
-        include: ['react-server-dom-webpack/client', 'react-dom/client'],
+        include: [
+          'react-server-dom-webpack/client',
+          'react-server-dom-webpack/client.edge',
+          'react-dom/client',
+        ],
```

Comment:
this does not seem to work. not sure if I am putting this in the correct spot yet